routes/postRoute.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express')
+const Router = express.Router()
+const multer = require('multer')
+
+// set destination for your files
+const upload = multer({dest:"uploads/"})
+
+//import post logic controllers
+const { createPost } = require('../controllers/postController')
+
+Router.route('/').post( upload.single("media_url"), createPost)
```

Comment:
please no authorization to know if user has permission to make the post? like a middleware to authorize the user.